workflow/rules/intersect/gdp-loss.smk

```diff
@@ -0,0 +1,67 @@
+"""Calculates the gdp losses through the storm damage.
+
+"""
+
+
+rule intersection_gdploss:
+    input:
```

Comment:
I think we can snakemake-ify the inputs and outputs.

Reply:
Are you referring to the lists? I use the lists to state that _all_ are required (rather than one through a wildcard). Maybe snakemake has a better way thought, happy to hear input on this

Reply:
updated and resolved